test/unit-and-integration/stake.test.ts

```diff
@@ -1,15 +1,18 @@
 import { cleanup, fireEvent, render, screen } from '@testing-library/svelte';
 import '@testing-library/jest-dom';
-import { dotApi, storeConnected, storeMsaInfo } from '../../src/lib/stores';
+import { dotApi, storeConnected } from '../../src/lib/stores';
 import Stake, { stakeAmount } from '$components/Stake.svelte';
 import { user } from '../../src/lib/stores/userStore';
+import { KeyringPair } from '@polkadot/keyring/types';
+import Keyring from '@polkadot/keyring';
+import { vi } from 'vitest';
 
 const mocks = vi.hoisted(() => {
   const resolvedApi = {
     isReady: vi.fn().mockResolvedValue(true),
   };
 
-  const mockApiPromise = vi.fn();
+  const mockApiPromise: any = vi.fn();
```

Comment:
rather than any, can we typecast it to ApiPromise so that we can avoid unnecessary any's